lib/lambda/update/updatePackage.ts

```diff
@@ -0,0 +1,217 @@
+import { response } from "libs/handler-lib";
+import { APIGatewayEvent } from "aws-lambda";
+import { getPackage, getPackageChangelog } from "libs/api/package";
+import { produceMessage } from "libs/api/kafka";
+import { ItemResult } from "shared-types/opensearch/main";
+import { events } from "lib/packages/shared-types";
+import { z } from "zod";
+
+const sendDeleteMessage = async (topicName: string, packageId: string) => {
+  await produceMessage(
+    topicName,
+    packageId,
+    JSON.stringify({
+      id: packageId,
+      deleted: true,
+      isAdminChange: true,
+      adminChangeType: "delete",
+    }),
+  );
+};
+
+const sendUpdateValuesMessage = async (
+  topicName: string,
+  packageId: string,
+  updatedFields: object,
+  changeMadeText: string,
+  changeReason?: string,
```

Comment:
I think it'd be easier to know what the arguments are when calling the function

Reply:
Could you please clarify this one?